lib/note-list/index.tsx

```diff
@@ -348,11 +348,6 @@ export class NoteList extends Component {
       this.recomputeHeights();
     }
 
-    // Ensure that the note selected here is also selected in the editor
-    if (selectedNoteId !== prevProps.selectedNoteId) {
-      onSelectNote(selectedNoteId);
-    }
```

Comment:
does this PR also fix the thrashing issue?

Reply:
It should if my hypothesis that this code is the cause. I am unable to replicate so it is a bit of a shot in the dark. I wasn't able to figure out any other reason for the thrashing to occur. I think once we get more things into redux we can reduce the number of events that modify the selected note this eliminating any additional cause.